src/renderer/components/editorWithTabs/editor.vue

```diff
@@ -675,10 +675,12 @@ export default {
     photoCreatorClick: (url) => {
       shell.openExternal(url)
     },
-    jumpClick: (linkInfo) => {
+    jumpClick (linkInfo) {
       const { href } = linkInfo
```

Comment:
I think no need to use the first `if statement` now? because we already handle url case in the `FORMAT_LINK_CLICK`.

BTW,

I prefer write in this way:

```js
this.$store.dispatch('FORMAT_LINK_CLICK', { data: { href }, dirname: window.DIRNAME })
```

Reply:
Thank you very much, I'll update tonight.

Reply:
thanks.